nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -130,6 +130,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 - The option [`services.nextcloud.config.dbport`] of the Nextcloud module was removed to match upstream.
   The port can be specified in [`services.nextcloud.config.dbhost`](#opt-services.nextcloud.config.dbhost).
 
+- With a bump to `sonarr` v4, existing config database files will be upgraded automatically, but the [upstream release announcement](https://forums.sonarr.tv/t/sonarr-v4-released/33089) reports that such upgrades have failed in the past when existing working configs were actually corrupted.
```

Comment:
How is this actionable to the user?

Reply:
It's really not actionable, other than a user choosing not to upgrade because they're worried, which I can't really imagine.

Perhaps this is better? Or we can just omit, it's not a big deal as far as I'm concerned.
```suggestion
- With a bump to `sonarr` v4, existing config database files will be upgraded automatically, but note that some old quietly-corrupted configs [might fail to upgrade cleanly](https://forums.sonarr.tv/t/sonarr-v4-released/33089).
```